sdk/Lusid.Sdk/Utilities/ClientCredentialsFlowTokenProvider.cs

```diff
@@ -125,17 +150,18 @@ private static async Task<AuthenticationToken> GetNewToken(ApiConfiguration apiC
                 // Send request
                 var response = await httpClient.SendAsync(tokenRequest);
                 var body = await response.Content.ReadAsStringAsync();
-
-                if (!response.IsSuccessStatusCode)
+                 
+                if ((int) response.StatusCode != 200)
                 {
                     throw new HttpRequestException(
                         $"Could not retrieve an authentication token from the specified identity provider. The request to {tokenRequest.RequestUri} returned an unsuccessful status code of {response.StatusCode} and the response body: {body}");
                 }
-
+                
                 var parsed = JsonConvert.DeserializeObject<Dictionary<string, string>>(body);
 
-                var apiToken = parsed["access_token"];                
+                var apiToken = parsed["access_token"];
                 var expires = parsed["expires_in"];
+                var refreshExpires = "5400"; // Refresh token expires in 90 minutes - Speak to Xan if you think this has changed
```

Comment:
promote this to a class variable that's readonly and make it an int so you don't have to do tryparse below